populate_course_run_reporting_type_and_announcement.py

```diff
@@ -0,0 +1,80 @@
+import argparse
```

Comment:
The only other automated task is the Transifex task built by @rlucioni. The rest are one-off scripts.